app/assets/stylesheets/font-awesome.scss

```diff
@@ -1,21 +0,0 @@
-$fa-font-path: "@fortawesome/fontawesome-free/webfonts";
-@import "@fortawesome/fontawesome-free/scss/fontawesome";
-
-@font-face {
-  font-family: 'Font Awesome 5 Free';
-  font-style: normal;
-  font-weight: 900;
-  font-display: $fa-font-display;
-  src: font-url('#{$fa-font-path}/fa-solid-900.eot');
-  src: font-url('#{$fa-font-path}/fa-solid-900.eot?#iefix') format('embedded-opentype'),
-  font-url('#{$fa-font-path}/fa-solid-900.woff2') format('woff2'),
-  font-url('#{$fa-font-path}/fa-solid-900.woff') format('woff'),
-  font-url('#{$fa-font-path}/fa-solid-900.ttf') format('truetype'),
-  font-url('#{$fa-font-path}/fa-solid-900.svg#fontawesome') format('svg');
```

Comment:
This is a Sprockets function, so... it's just being compiled as-is currently.

I _think_ we might be relying on the benevelence of browser caches?